client/my-sites/domains/domain-management/transfer/transfer-out/index.jsx

```diff
@@ -102,7 +102,11 @@ class Transfer extends React.Component {
 	};
 
 	isDataLoading() {
-		return ! this.props.wapiDomainInfo.hasLoadedFromServer || this.props.isRequestingSiteDomains;
+		return (
+			! this.props.wapiDomainInfo.hasLoadedFromServer ||
+			this.props.isRequestingSiteDomains ||
+			! this.props.hasSiteDomainsLoaded
```

Comment:
This fixes an unrelated bug that caused this whole screen to go blank when we're loading from empty/fresh state.